_posts/blog/2016-09-06-hack-week.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "upday Hack Week"
+description: Three days, 49 participants, 30+ ideas.
+modified:
+categories: blog
+author: florina_muntenescu
+excerpt: Three days, 49 participants and 30+ ideas. Find out what we did during our Hack-Week, why and what did we learn from it.
+tags: [Development, Hackathon, Hack-Week]
+date: 2016-09-06T15:39:55-04:00
+---
+
+After two smaller hackathons: one for Android developers, dedicated to testing; and the other one for the entire engineering, targeted to improving our continuous integration system, we decided that it's time to go to the next level. We wanted to expand the innovation opportunity outside the engineering teams so we created our own version of a Hack-Week.
+
+Let's discover together who participated in this, how were the ideas pitched, who worked on implementing them and most of all what we learned from all of this.
```

Comment:
>Let's discover together who participated in this, how were the ideas pitched, who worked on implementing them and most of all what we learned from all of this.

This sentence is a bit much for the content. It's not really a journey of discover, more an explanation: a statement of facts.